docs/develop/terrain/using-terrain-testnet.md

```diff
@@ -1,40 +1,41 @@
-# Use Terrain with the testnet
+# Use Terrain with testnet
 
 The [Pisco testnet](../../learn/glossary.md#testnet) is used for testing transactions on the Terra network.
 
 ## Prerequisites
 
-- [Install the Terra Station browser extension](../../learn/terra-station/download/terra-station-chrome.md)
+- Install the [Terra Station Chrome extension](../../learn/terra-station/download/terra-station-chrome.md).
 
-## Create a Pisco wallet
+## Create a test wallet
 
-[Create a new wallet](../../learn/terra-station/download/terra-station-chrome.md#create-a-wallet) using the Terra Station extension. It's recommended that you name this wallet "Pisco" or "testnet" so it's easy to remember.
+Open the Terra Station browser extension and [create a new wallet](../../learn/terra-station/download/terra-station-chrome.md#create-a-wallet) for testing purposes.
 
-After creating a Pisco wallet and storing the seed phrase, request funds from the testnet faucet:
+:::{tip}
+Name your test wallet something that's easy to remember such as "testnet" or "pisco".
+:::
 
-https://faucet.terra.money
+After creating a testing wallet and storing the seed phrase, you may request funds from [the testnet faucet](https://faucet.terra.money) in order to have tokens to carry out transactions on testnet.
 
-:::{danger}
-Make sure you have your seed phrase stored somewhere since you will need it to complete this tutorial.
+:::{Warning}
+Make sure you have your seed phrase available as you will need it to complete this tutorial.
 :::
```

Comment:
@evanorti you think we should combine the tip and warning messages?  or is this fine.

Reply:
I think this is fine for now